docs/man.rst

```diff
@@ -26,178 +26,154 @@ To install the Quipucords server and command line interfaced (CLI) client:
+  Determines whether to open the ``port`` in the firewall during the installation. This option enables communication between the Quipucords server and any remote clients over the port defined in ``port``. Contains a true or false value. Defaults to ``true``. Supply ``false`` to install without opening the server port in the firewall. The installation script must run with elevated privileges to open the server port.
 
-``-e  dbms_user=postgres``
+``--dbms-user=DBMS_USER``
 
   Specifies the database user for PostgreSQL. Defaults to ``postgres``.
 
-``-e dbms_password=pass123``
+``--dbms-password=DBMS_PASSWORD``
 
   Specifies the database password for PostgreSQL. Defaults to ``password``.
 
-``-e server_username=adminid1``
+``--username=SERVER_USERNAME``
 
   Sets the Quipucords server user name. Defaults to ``admin``.
 
-``-e server_password=adminpw1``
+``--password=SERVER_PASSWORD``
 
   Sets the Quipucords server password. Defaults to ``qpcpassw0rd``.
 
-``-e server_user_email=adminid1@example.com``
-
-  Sets the Quipucords server user email address. Defaults to ``admin@example.com``.
-
-``-e use_supervisord=false``
-
-  Controls whether to start the Quipucords server with supervisord. Contains a true or false value. Defaults to ``true``. Supply ``false`` to start the server without supervisord.
-
-``-e home_dir=~/home_dir``
-
-  Sets the fully qualified path to the installation directory for the Quipucords server. Defaults to ``~/quipucords/``.
-
-``-e offline_files=~/offline``
-
-  Sets the fully qualified path to the files needed to complete an offline installation. Required if ``install_offline=true``.
-
-``-e server_name=quipucords2``
 
-  Sets the name for the Quipucords server. Defaults to ``quipucords``.
-
-``-e server_http_timeout=90``
-
-  Sets the HTTP timeout length for the Quipucords server. Defaults to ``120``.
-
-``-e inspect_job_timeout=10700``
-
-  Sets the network inspection scan timeout in seconds. Defaults to ``10800`` (3 hours).
-
-``-e connect_job_timeout=500``
-
-  Sets the network connection scan timeout in seconds. Defaults to ``600`` (10 minutes).
-
-``-e ansible_log_level=10``
-
-  Sets the level of log output by Ansible during network scans. Defaults to ``0``, which is the value for no extended logs.
-
-Installing offline
-------------------
+Installing the server offline
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 If you choose the offline option to run the install command, you must do the following steps:
 
 #. Obtain the installation packages on a machine with internet connectivity.
+  - Download the following server image: https://github.com/quipucords/quipucords/releases/latest/download/quipucords_server_image.tar.gz
```

Comment:
The offline section numbering is off for the server - it goes 1. 1. 2.

Reply:
Fixed.